### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge.turrado@docplanner.com>

Related https://github.com/kedacore/keda/issues/2978

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]

File: adapter/generated/openapi/zz_generated.openapi.go [hunk 1]
```diff
@@ -4997,11 +4997,11 @@ func schema_k8sio_api_core_v1_NodeAddress(ref common.ReferenceCallback) common.O
...
+							Description: "Node address type, one of Hostname, ExternalIP or InternalIP.",
```
Comment (tomkerkhove) [inline adapter/generated/openapi/zz_generated.openapi.go:5000]: Are these ok?

Comment (JorTurFer) [reply adapter/generated/openapi/zz_generated.openapi.go:5000]: yes, they are autogenerated files based on the code+dependencies

**Pull request merged**